clang/include/clang/AST/ExprCXX.h

```diff
@@ -3048,10 +3050,12 @@ class OverloadExpr : public Expr {
 
       Result.HasFormOfMemberPointer = (E == Ovl && Ovl->getQualifier());
       Result.IsAddressOfOperand = true;
+      Result.IsAddressOfOperandWithParen = HasParen;
```

Comment:
Cleanup nit: I would vastly prefer FindResult have a constructor & deleted default constructor.  This seems like it would be really easy to forget to initialize on some branch.

Reply:
I added a default member initializer for everything